src/orka-builder.ts

```diff
@@ -167,8 +166,13 @@ export default class OrkaBuilder {
     return this;
   }
 
-  withGrowthbook() {
-    this.queue.push(() => growthbook(this.config));
+  loadGrowthbookFeatures() {
+    this.queue.push(() => {
+      const { createGrowthbook } = require('./initializers/growthbook');
+      const gb = createGrowthbook(this.config.growthbook);
+      if (!gb) return;
+      gb.loadFeatures().catch(e => getLogger('orka.growthbook').error('Unable to load features', e));
```

Comment:
🙃 You can use await here. It will delay startup by a few ms but even during startup you won't ever get a wrong FF. WDYT?